### ISSUE TITLE
Reproduce vgg face result.

### ISSUE THREAD

**Issue by ildoonet** [ildoonet/deepface#1]
Mainly, Two things should be tested and implemented.

- After getting bounding boxes using dlib, what is the proper way to crop roi regions?
- Evaluation Codes for public datasets are needed.

**Comment by ildoonet** [comment]
without embedding loss and training on LFW dataset, achieved 0.94 1-eer, which is sighthly lower than the result from the paper that used embedding loss & finetuning on LFW dataset.
